package.json

```diff
@@ -345,17 +345,17 @@
     "@glif/filecoin-wallet-provider": "2.0.0-alpha.14",
     "@ledgerhq/hw-app-eth": "6.24.1",
     "@ledgerhq/hw-transport-webhid": "6.24.1",
+    "@solana/web3.js": "1.36.0",
     "@types/jszip": "^3.1.6",
     "@types/parse-torrent": "^5.8.3",
     "@types/react-router-dom": "^5.1.9",
     "@types/webtorrent": "^0.109.0",
-    "@solana/web3.js": "1.36.0",
     "array-move": "^2.2.1",
-    "bignumber.js": "^9.0.2",
+    "bignumber.js": "9.0.2",
```

Comment:
we probably want to pin this to the version that is currently used in package-lock.json rather than `9.0.2` specifically if it's a downgrade

Reply:
The version in package-lock.json is `9.0.2`